### ISSUE TITLE
Binding address from Env

### ISSUE THREAD

**Issue by andreasolund** [Scorpiion/Vane#20]
Vane doesn't support specifying the binding address since it's hard coded to `127.0.0.1` in [serve.dart#L37](https://github.com/DartVoid/Vane/blob/master/lib/src/serve.dart#L37). I propose that we add the `ADDRESS` environment variable and change the default address to `InternetAddress.LOOPBACK_IP_V4`.

Basically something like this:
```dart
var addressEnv = Platform.environment['ADDRESS'];
var address = addressEnv == null ? InternetAddress.LOOPBACK_IP_V4 : addressEnv;
```

**Comment by Scorpiion** [comment]
Now "0.0.0.0" is the default address, the address can also be set either via an optional parameter to serve() or as an environment variable. Same goes for port.

Fixed in commit 85849b281d4fc7d9c632ef157b44c2264f9e5b59.

**Comment by andreasolund** [comment]
:+1:
